std_msgs/Header header
nav_msgs/OccupancyGrid grid_map
float64 height_scale                # scaling factor of values in height grid map
